graphistry/compute/predicates/is_in.py

```diff
@@ -1,21 +1,106 @@
 from typing import Any, List
 import pandas as pd
+import numpy as np
+from datetime import datetime, date, time
 
 from graphistry.utils.json import assert_json_serializable
 from .ASTPredicate import ASTPredicate
+from .temporal_values import TemporalValue, DateTimeValue, DateValue, TimeValue
+from ._normalization import normalize_isin_value
 from graphistry.compute.typing import SeriesT
 
 
 class IsIn(ASTPredicate):
     def __init__(self, options: List[Any]) -> None:
-        self.options = options
```

Comment:
this value parsing feels like it should be factored out as other temporal operators can use them too